repose-aggregator/src/docs/asciidoc/filters/tenant-culling-filter.adoc

```diff
@@ -0,0 +1,52 @@
+= Tenant Culling Filter
+
+The Tenant Culling filter is used on resources that need further specific tenant id information.
+The filter looks at all the tenants that Keystone returns for a user to creates and populates a list of them based on the specific roles that were used to grant access to the resource.
+
+== General filter information
+* *Name:* tenant-culling
+* *Default Configuration:* No configuration
+* *Bundle:* repose-filter-bundle
```

Comment:
do we put initial version in this section?

Reply:
Updated with a place holder since we don't know exactly which version this will actually be brought in to. Also added a tasks to the release stories to do this:

* [v8.6.?.?[(https://repose.atlassian.net/browse/REP-6036)
* [v8.?.?.?](https://repose.atlassian.net/browse/REP-6035)